features/gba-dev/gezagsverhouding-gba.feature

```diff
@@ -0,0 +1,142 @@
+
+    Voorbeelden:
+    | gba in onderzoek waarde |
+    | 110000                  |
+    | 113200                  |
+    | 113210                  |
+
+  Abstract Scenario: 'indicatie curateleregister (33.10)' is in onderzoek
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'gezagsverhouding' gegevens
+    | naam                                 | waarde                    |
+    | aanduiding in onderzoek (83.10)      | <gba in onderzoek waarde> |
+    | datum ingang onderzoek (83.20)       | 20020701                  |
+    | indicatie curateleregister (33.10)   | 1                         |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | indicatieCurateleRegister       |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                                              | waarde                    |
+    | indicatieCurateleRegister                         | true                      |
+    | gezagInOnderzoek.aanduidingGegevensInOnderzoek    | <gba in onderzoek waarde> |
+    | gezagInOnderzoek.datumIngangOnderzoek             | 20020701                  |
+
+    Voorbeelden:
+    | gba in onderzoek waarde |
+    | 110000                  |
+    | 113300                  |
+    | 113310                  |
+
+  Abstract Scenario: '<in onderzoek veld>' is in onderzoek, maar wordt niet gevraagd
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'gezagsverhouding' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | <gba in onderzoek waarde> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    | <in onderzoek veld>             | <waarde>                  |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | <field>                         |
+    Dan heeft de response een persoon zonder gegevens
+
+    Voorbeelden:
+    | gba in onderzoek waarde | in onderzoek veld                    | waarde | field                      |
+    | 113200                  | indicatie gezag minderjarige (32.10) | 1      | indicatieCurateleRegister  |
+    | 113210                  | indicatie gezag minderjarige (32.10) | 2      | indicatieCurateleRegister  |
+    | 113300                  | indicatie curateleregister (33.10)   | 1      | indicatieGezagMinderjarige |
+    | 113310                  | indicatie curateleregister (33.10)   | 1      | indicatieGezagMinderjarige |
+
+  Rule: Onjuiste gegevens worden niet geleverd.
```

Comment:
Ik snap deze rule niet in combinatie met de scenario hieronder. Zoals ik het begrijp werkt onjuist als volgt:
- in de rij met volg_nr 0 krijgt kolom onjuist_ind de waarde 'O'
- de rij met volg_nr 0 wordt opgehoogd naar volg_nr 1
- een nieuwe rij met volg_nr 0 wordt toegevoegd

Als je in de onderstaande scenario wilt zeggen dat de betreffende persoon gecorrigeerd moet worden naar 'geen gezagsverhouding', dan zou de gegeven stappen er volgens mij als volgt eruit moeten zien
```
    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'gezagsverhouding' gegevens
    | naam                                 | waarde   |
    | indicatie gezag minderjarige (32.10) | 12       |
    | datum van opneming (86.10)           | 20080706 |
    En de 'gezagsverhouding' is gecorrigeerd naar de volgende gegevens
    | naam                       | waarde   |
    | datum van opneming (86.10) | 20080707 |

```

Reply:
Aangepast